src/manifest.c

```diff
@@ -1434,7 +1434,12 @@ static int is_version_data(const char *filename)
 {
 	if (strcmp(filename, "/usr/lib/os-release") == 0 ||
 	    strcmp(filename, "/usr/share/clear/version") == 0 ||
-	    strcmp(filename, "/usr/share/clear/versionstamp") == 0) {
+	    strcmp(filename, "/usr/share/clear/versionstamp") == 0 ||
```

Comment:
For the future:
Once we have more than a couple of files, it would be better to loop over an array of strings, rather than individual strcmp based compares. Whilst it is true that gcc has code to optimize compares against constant strings, I don't think this will help in this case.